### PR TITLE
CSS Ecommerce Page - Michael Van Doorn

### PR DESCRIPTION
**Pull request by Michaelvxd**
CleanShot 2022-12-30 at 19 56 44@2x

### CODE REVIEW

File: css-ecommerce-page/styles.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.row {
+  display: flex;
+  justify-content: center;
+  margin: 0 100px;
+}
+
+.col {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
```
Comment (Michaelvxd) [inline css-ecommerce-page/styles.css:23]: I feel like there's a better way to make images scale. Am I missing something?

File: css-ecommerce-page/styles.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+img {
+  width: 100%;
+  height: 100%;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  margin: 0 100px;
```
Comment (Michaelvxd) [inline css-ecommerce-page/styles.css:17]: Could I set the margins on the left/right of the website in a better manner?

Comment (scrumptiousdev) [reply css-ecommerce-page/styles.css:17]: It really depends on website needs — if one of many components / sections needs full width then setting margins on each component (like you can add new style `max-width-1200` class and add it to `row` div whenever it needs the margins on the sides. But on the other hand, if there is no full-width component, then you can set margins on `.container` or `body` itself

File: css-ecommerce-page/styles.css (hunk 1 above)
Comment (scrumptiousdev) [reply css-ecommerce-page/styles.css:23]: It is not too bad how you have it right now but when you set both image's height and width 100% then it stretches the image — if you take a look a the Louise Vuitton image it's stretched atm. You can either use backgroun-image css or object-fit

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

File: css-ecommerce-page/styles.css (hunk 1 above)
Comment (Michaelvxd) [reply css-ecommerce-page/styles.css:24]: Updated! Ty

**Review by scrumptiousdev — APPROVED** [review]
Approved!

**Pull request merged**